tests/test_confidence.py

```diff
@@ -0,0 +1,87 @@
+from vulture import Vulture
+
+from . import skip_if_python_2
+from . import skip_if_python_3
+
+
+def check_min_confidence(code, min_conf, linenos, confidences):
```

Comment:
This design is suboptimal since callers have to make sure that both lists have the same length. Better pass a aingle list of pairs.